src/clr-core/alert/alert.element.ts

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import '@clr/core/button';
+import {
+  baseStyles,
+  CommonStringsService,
+  DefaultSizes,
+  event,
+  EventEmitter,
+  property,
+  registerElementSafely,
+  StatusTypes,
+} from '@clr/core/common';
+import {
+  checkCircleIcon,
+  ClarityIcons,
+  exclamationCircleIcon,
+  exclamationTriangleIcon,
+  infoCircleIcon,
+  timesIcon,
+} from '@clr/core/icon-shapes';
+import { html, LitElement } from 'lit-element';
+import { styles } from './alert.element.css';
+
+const iconMap = {
+  info: {
+    shape: ClarityIcons.getIconNameFromShape(infoCircleIcon),
+    title: CommonStringsService.keys.info,
+  },
+  success: {
+    shape: ClarityIcons.getIconNameFromShape(checkCircleIcon),
+    title: CommonStringsService.keys.success,
+  },
+  warning: {
+    shape: ClarityIcons.getIconNameFromShape(exclamationTriangleIcon),
+    title: CommonStringsService.keys.warning,
+  },
+  danger: {
+    shape: ClarityIcons.getIconNameFromShape(exclamationCircleIcon),
+    title: CommonStringsService.keys.danger,
+  },
+};
+
+/**
+ * Web component alerts.
+ *
+ * ```typescript
+ * import '@clr/core/alert';
+ * ```
+ *
+ * ```html
+ * <cwc-alert app-level closable="false" status="warning">
+ *    <cwc-alert-text>Lorem ipsum dolor sit amet</cwc-alert-text>
+ *    <cwc-alert-actions>
+ *       <cwc-button>Fix</cwc-button>
+ *    </cwc-alert-actions>
+ * </cwc-alert>
+ * ```
+ *
+ * @noInheritDoc
+ * @beta 3.0
+ * @element cwc-alert
```

Comment:
Once the sass is updated can we add the @cssprop decorators here for the auto docs?